builder/qemu/config.go

```diff
@@ -502,7 +502,23 @@ type Config struct {
 	// `virtio-scsi`. The Qemu builder uses `virtio` by default.
 	// Some ARM64 images require `virtio-scsi`.
 	CDROMInterface string `mapstructure:"cdrom_interface" required:"false"`
-
+	// Use a virtual (emulated) TPM device to expose to the VM.
+	VTPM bool `mapstructure:"vtpm" required:"false"`
+	// Use version 1.2 or 2.0 of the TPM specification for the emulated TPM.
+	//
+	// Default value is 2.0, to use version 1.2 of the specs, set to false.
+	VTPMUseTPM2 config.Trilean `mapstructure:"use_tpm2" required:"false"`
```

Comment:
I think this shouldn't be a Trilean, I think you need to know the difference between unset and false here so thats why you used one but I think you could change this to be a value which turns on TPM 1.2, and if its set to false just to use TPM 2.0, that way the "default" behavior of the flag being false doesn't require extra logic to interpolate, maybe like VTPMUseTPM1, what do you think?

Reply:
Yeah, I made that same remark to myself while reading through the code, and you're right, I guess we can reverse the option and make it a regular bool rather than a Trilean.
I'll amend this quickly.